core/src/toga/widgets/activityindicator.py

```diff
@@ -19,8 +20,9 @@ def __init__(
             will be applied to the widget.
         :param running: Describes whether the indicator is running at the
             time it is created.
+        :param kwargs: Initial style properties.
         """
-        super().__init__(id=id, style=style)
+        super().__init__(id, style, **kwargs)
```

Comment:
On this (and all other widgets), there's a need to verify with a test that `**kwargs` is actually being passed through.